### PR TITLE
CDR switches cleanup

### PR DESCRIPTION
**Pull request by katarkow**
## Purpose of this PR

Documentation update and removing switch `cm_DAC_eff`, which is not used anymore.

### CODE REVIEW

**Comment by LaviniaBaumstark** [comment]
please add `cm_DAC_eff` to the list of outdated switches.

File: scripts/start/readCheckScenarioConfig.R [hunk 1]
```diff
@@ -140,6 +140,7 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
        "cm_OILRETIRE" = "Now always on by default, see https://github.com/remindmodel/remind/pull/1102",
        "cm_fixCO2price" = "Was never in use, removed in https://github.com/remindmodel/remind/pull/1369",
        "cm_calibration_FE" = "Deleted, only used for old hand made industry trajectories, see https://github.com/remindmodel/remind/pull/1468",
+       "cm_DAC_eff" = "Deleted, not used anymore, see https://github.com/remindmodel/remind/pull/1487",
```
Comment (katarkow) [inline scripts/start/readCheckScenarioConfig.R:143]: @LaviniaBaumstark does that add the removed switch to outdated switches?

Comment (LaviniaBaumstark) [reply scripts/start/readCheckScenarioConfig.R:143]: yes

**Comment by orichters** [comment]
I created a merge conflict that I fixed here…

**Pull request merged**